binary/Dockerfile.linux-binary

```diff
@@ -0,0 +1,27 @@
+# This image runs the dcos-cli test suite.
+
+FROM ubuntu:15.04
+MAINTAINER support@mesosphere.com
+
+RUN apt-get update && apt-get install -y \
+  httpie \
+  jq \
+  make \
+  openssh-client \
+  git \
+  sudo \
+&& sudo apt-get update --fix-missing \
+&& sudo apt-get install -y python-dev build-essential \
+&& sudo apt-get install -y python-pip python-virtualenv \
+&& pip install pip --upgrade \
+&& pip install pyinstaller oauth2client
```

Comment:
We can make this part of the `requirement.txt` and we get it for free when do you do `make env`, no?